cardano-config/src/Cardano/Config/Logging.hs

```diff
@@ -227,6 +231,11 @@ loggingCardanoFeatureInit disabled' conf = do
   Cardano.BM.Backend.Monitoring.plugin logConfig trace switchBoard
       >>= loadPlugin switchBoard
 
+#if defined(linux_HOST_OS)
+  Cardano.BM.Scribe.Systemd.plugin logConfig baseTrace switchBoard
```

Comment:
'baseTrace' should be 'trace' so it can be turned on/off.

Reply:
@CodiePP, done.